### PR TITLE
Add header styles 

### PR DESCRIPTION
**Pull request by yararefaee**

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.HeaderSecion{
+    display: flex;
+
```
Comment (ziadbastawy) [inline src/components/Header/Header.css]: please, remove empty line

File: src/components/Header/Header.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.HeaderSecion{
+    display: flex;
+   
+}
+
+.icons{
+    margin-top: 10px;
+
```
Comment (ziadbastawy) [inline src/components/Header/Header.css]: here also

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+.icon {
+    flex: 1;
+    align-items: right;
+    display: inline-block;
+    margin-left: 2px;
+ }
+
+ .Search {
+    margin: 0 auto;
+
```
Comment (ziadbastawy) [inline src/components/Header/Header.css]: here also

File: src/components/Header/Header.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+     margin-top: 15px;
+    border-radius: 2px;
+    background-color: rgb(250, 249, 249);
+    border: rgba(80, 80, 80, 0.623) 1px solid;
+    align-content: center;
+    padding: 0 auto;
+    text-align: center;
+    
+
+
```
Comment (ziadbastawy) [inline src/components/Header/Header.css]: here also

File: src/components/Header/Header.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+ }
+ .instaLogo {
+
+     margin-left: 20px;
+    align-self: flex-start;
+    
+
+
```
Comment (ziadbastawy) [inline src/components/Header/Header.css]: here also

File: src/components/Header/Header.css [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    
+
+   
+ }
+
+ .avatarIcon {
+     border: double;
+     border-radius: 15px;
+     margin-right: 10px;
+ }
```
Comment (ziadbastawy) [inline src/components/Header/Header.css]: please, add emplty line

File: src/components/Header/Header.jsx [hunk 7]
```diff
@@ -1,9 +1,33 @@
...
+    return (<div className="HeaderSecion">
+        <div className="instaLogo">
+            <img src={InstaLogo} alt="Logo" width="120px" />
+        </div>
+        <div className="Search">
+            <input type="text" className="SearchBar" placeholder="search..."></input>
+        </div>
+        <div className="icons">
+        <a href="" target="self">
+            <img src={Home} className="icon" alt="Home" width="23px" /></a>
```
Comment (ziadbastawy) [inline src/components/Header/Header.jsx]: please, format the file and remove empty line

File: src/components/Header/Header.css [hunk 8]
```diff
@@ -0,0 +1,49 @@
+.HeaderSecion{
+    display: flex;
+   
+}
+
+.icons{
```
Comment (majidzeno) [inline src/components/Header/Header.css]: Please name this class following PascalCase , it should be **Icons**

File: src/components/Header/Header.css [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    margin-top: 10px;
+
+}
+
+.icon {
```
Comment (majidzeno) [inline src/components/Header/Header.css]: Please name this following PascalCase , it would be **Icon**

File: src/components/Header/Header.css [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    background-color: rgb(250, 249, 249);
+    border: rgba(80, 80, 80, 0.623) 1px solid;
+    align-content: center;
+    padding: 0 auto;
+    text-align: center;
+    
+
+
+ }
+ .instaLogo {
```
Comment (majidzeno) [inline src/components/Header/Header.css]: Please follow PascalCase naming, this would be **InstaLogo**

File: src/components/Header/Header.css [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+     margin-left: 20px;
```
Comment (majidzeno) [inline src/components/Header/Header.css:32]: please align all styles, format the document before submitting the code

File: src/components/Header/Header.css [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+    align-self: flex-start;
+    
+
+   
+ }
+
+ .avatarIcon {
```
Comment (majidzeno) [inline src/components/Header/Header.css]: Please follow PascalCase

**Review by majidzeno — APPROVED** [review]

**Pull request merged**